### PR TITLE
Add ZoneH CSV parser bot + RewindableFileHandle

### PR DESCRIPTION
**Pull request by chorsley**
Adds a parser for ZoneH defacement reports distributed via CSV. Also adds a RewindableFileHandle class which may be useful for easily obtaining raw data during file / CSV parsing.

### CODE REVIEW

File: intelmq/bots/parsers/zoneh/parser.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import csv
+import io
+from urllib.parse import urlparse
+
+from intelmq.lib import utils
+from intelmq.lib.bot import ParserBot, RewindableFileHandle
+from intelmq.lib.message import Event
+
+
+class ZoneHParserBot(ParserBot):
```
Comment (wagner-certat) [inline intelmq/bots/parsers/zoneh/parser.py]: You are using the ParserBot here and below you are overwriting it's `process` method. So you can use the plain `Bot` here.

**Comment by wagner-certat** [comment]
What is the URL to the Feed actually? Just fixing some issues here (incl documentation) and I'd like to add that. And what is a reasonable rate limit?

**Comment by chorsley** [comment]
Firstly, apologies taking so long with the PR changes you've requested.

Regarding the data source, it's not a URL feed, but is using the mail collector bot. The reports are sent as CSV attachments. The feed is available on request as far as I know.

**Comment by wagner-certat** [comment]
Okay thanks. I will send you a PR with the modifications I have

**Comment by wagner-certat** [comment]
See https://github.com/chorsley/intelmq/pull/1

**Comment by SYNchroACK** [comment]
@wagner-certat I talked with @chorsley to merge your pull req on his forked repo. If you have a chance, review the current status of this pull (Im a little bit lost).

**Pull request merged**